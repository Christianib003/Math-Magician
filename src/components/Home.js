import React from 'react';

const Home = () => (
  <>
    <div className="welcome">
      <h2 className="hello">welcome to our page!</h2>
      <p className="text">
        But I must explain to you how all this mistaken idea of praising pain.
        I will give you a complete account of the system, and expound teachings of the great
        the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself
        but because those who do not know to pursue pleasure consequences that are painful.
        Nor again is there anyone who loves or pursues or desires to obtain but because occasionally
        circumstances occur in which toil and pain can procure him some great pleasure.
        To take a trivial example, which of us ever undertakes laborious physical exercise,
        except to obtain some advantage from it? But who has right to find fault with a man
      </p>
      <p className="text">
        On the other hand, we denounce with righteous indignation
        and dislike men who are so beguiled and demoralized by the
        charms of pleasure of the moment, so blinded by desire,
        that they cannot foresee the pain and trouble ; and equal
        blame belongs to those who fail in their duty through weakness of will.
        as saying through shrinking from toil and pain. In a free hour,
        when our power of choice is untrammelled being able to do what we like best,
        every pleasure is to be welcomed of duty or the obligations of business it will
        frequently occur The wise man therefore always holds in these matters to this
        principle of selection: he rejects pleasures to secure other greater pleasures
      </p>
    </div>
  </>
);

export default Home;
