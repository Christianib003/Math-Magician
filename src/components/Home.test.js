import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toTree();
  expect(tree).toMatchSnapshot();
});
