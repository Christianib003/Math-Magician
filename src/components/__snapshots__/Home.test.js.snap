// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="hello"
        >
          welcome to our page!
        </h2>,
        <p
          className="text"
        >
          But I must explain to you how all this mistaken idea of praising pain. I will give you a complete account of the system, and expound teachings of the great the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself but because those who do not know to pursue pleasure consequences that are painful. Nor again is there anyone who loves or pursues or desires to obtain but because occasionally circumstances occur in which toil and pain can procure him some great pleasure. To take a trivial example, which of us ever undertakes laborious physical exercise, except to obtain some advantage from it? But who has right to find fault with a man
        </p>,
        <p
          className="text"
        >
          On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble ; and equal blame belongs to those who fail in their duty through weakness of will. as saying through shrinking from toil and pain. In a free hour, when our power of choice is untrammelled being able to do what we like best, every pleasure is to be welcomed of duty or the obligations of business it will frequently occur The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures
        </p>,
      ],
      "className": "welcome",
    },
    "rendered": Array [
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": "welcome to our page!",
          "className": "hello",
        },
        "rendered": Array [
          "welcome to our page!",
        ],
        "type": "h2",
      },
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": "But I must explain to you how all this mistaken idea of praising pain. I will give you a complete account of the system, and expound teachings of the great the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself but because those who do not know to pursue pleasure consequences that are painful. Nor again is there anyone who loves or pursues or desires to obtain but because occasionally circumstances occur in which toil and pain can procure him some great pleasure. To take a trivial example, which of us ever undertakes laborious physical exercise, except to obtain some advantage from it? But who has right to find fault with a man",
          "className": "text",
        },
        "rendered": Array [
          "But I must explain to you how all this mistaken idea of praising pain. I will give you a complete account of the system, and expound teachings of the great the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself but because those who do not know to pursue pleasure consequences that are painful. Nor again is there anyone who loves or pursues or desires to obtain but because occasionally circumstances occur in which toil and pain can procure him some great pleasure. To take a trivial example, which of us ever undertakes laborious physical exercise, except to obtain some advantage from it? But who has right to find fault with a man",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "nodeType": "host",
        "props": Object {
          "children": "On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble ; and equal blame belongs to those who fail in their duty through weakness of will. as saying through shrinking from toil and pain. In a free hour, when our power of choice is untrammelled being able to do what we like best, every pleasure is to be welcomed of duty or the obligations of business it will frequently occur The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures",
          "className": "text",
        },
        "rendered": Array [
          "On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble ; and equal blame belongs to those who fail in their duty through weakness of will. as saying through shrinking from toil and pain. In a free hour, when our power of choice is untrammelled being able to do what we like best, every pleasure is to be welcomed of duty or the obligations of business it will frequently occur The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  "type": [Function],
}
`;
